src/Symfony/Bundle/FrameworkBundle/Tests/Command/AboutCommand/AboutCommandTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Tests\Command\AboutCommand;
+
+use Symfony\Bundle\FrameworkBundle\Command\AboutCommand;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\FrameworkBundle\Tests\Command\AboutCommand\Fixture\TestAppKernel;
+use Symfony\Bundle\FrameworkBundle\Tests\TestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\Filesystem\Exception\IOException;
+use Symfony\Component\Filesystem\Filesystem;
+
+class AboutCommandTest extends TestCase
+{
+    /** @var TestAppKernel */
+    private $kernel;
+    /** @var Filesystem */
+    private $fs;
+
+    protected function setUp(): void
+    {
+        $this->fs = new Filesystem();
+        $this->kernel = new TestAppKernel('test', true);
```

Comment:
You should create these 2 vars in each test as here, you could leak memory (the alternative would be to unset those in teardown, but I think it's cleaner to just do everything in each test method as the duplicate code is simple enough).

Reply:
Done.